### PR TITLE
Clowder spec changes

### PR DESCRIPTION
**Pull request by bennyturns**
Signed-off-by: bennyturns <bturner@redhat.com>

## Secure Coding Practices Checklist Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [ ] Input Validation

- [ ] Output Encoding

- [ ] Authentication and Password Management

- [ ] Session Management

- [ ] Access Control

- [ ] Cryptographic Practices

- [ ] Error Handling and Logging

- [ ] Data Protection

- [ ] Communication Security

- [ ] System Configuration

- [ ] Database Security

- [ ] File Management

- [ ] Memory Management

- [ ] General Coding Practices

### CODE REVIEW

**Comment by SteveHNH** [comment]
approved!

File: deploy/clowdapp.yaml [hunk 1]
```diff
@@ -109,6 +56,62 @@ objects:
...
+          requests:
+            cpu: 200m
+            memory: 256Mi
+        volumes:
+        - emptyDir: {}
+          name: tmpdir
+        volumeMounts:
+        - mountPath: /tmp
+          name: tmpdir
+      objectStore:
```
Comment (psav) [inline deploy/clowdapp.yaml:113]: object store should be dedented

File: deploy/clowdapp.yaml [hunk 2]
```diff
@@ -109,6 +56,62 @@ objects:
     - replicas: 3
       partitions: 3
       topicName: platform.upload.resource-optimization
+    webServices:
```
Comment (psav) [inline deploy/clowdapp.yaml:59]: `webServices` needs to be inside deployments

**Comment by SteveHNH** [comment]
we merged another PR that does this fun stuff

**Pull request closed without merging**